go/host/node/node.go

```diff
@@ -545,31 +546,33 @@ func (a *Node) handleRoundWinner(result common.BlockSubmissionResponse) func() {
 		if err != nil {
 			log.Panic("could not determine round winner. Cause: %s", err)
 		}
-		if isWinner {
```

Comment:
I didn't actually make any changes here. I just flipped the logic so that it returns early if it's not a winner, rather than creating a huge `if` block.